src/plugins/cache/README.md

```diff
@@ -5,7 +5,7 @@
 - infos/provides =
 - infos/recommends =
 - infos/placements = pregetcache postgetcache
-- infos/status = maintained unittest shelltest specific global
+- infos/status = maintained unittest shelltest specific nodep global
```

Comment:
Except of (maybe) this change, I think most of these fixes from the auto heuristics do not improve the situation or are even wrong.

E.g. ccode is afaik configurable, it supports /escape and /chars. It only misses the docu how to do it, so probably it should have nodoc. I wouldn't waste too much time on reclassify everything, as this is planned in #3590 anyway.

Reply:
Then I will revert these auto-generated changes to avoid errors.